### PR TITLE
javascript-loops-practice exercise.-Dylan De Vei

### PR DESCRIPTION
**Pull request by dylandevei**
https://user-images.githubusercontent.com/99229316/160071484-afa1702b-90ea-4225-8b94-826491cee68b.mov

### CODE REVIEW

File: javascript-loops-practice/find-index.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/* exported findIndex */
+
+function findIndex(array, value) {
+  for (var i = 0; i < array.length; i++) {
+    if (array[i] === value) {
+      return i;
+    }
+  } if (value > array.length) {
```
Comment (codingcodymiller) [inline javascript-loops-practice/find-index.js]: Why `value > array.length`? Is this function checking if the `value` given is larger than the length of the array to search?

File: javascript-loops-practice/includes-seven.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+/* exported includesSeven */
+function includesSeven(array) {
+  var sevenArray = [];
+  for (var i = 0; i < array.length; i++) {
+    if (array[i] === 7) {
+      sevenArray.push(i);
+    }
+  } if (sevenArray > 0) {
```
Comment (codingcodymiller) [inline javascript-loops-practice/includes-seven.js]: I believe you meant `sevenArray.length`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Dylan! In addition to my other comments, please move all your return statements to their own line, you should never have a return statement on the same line as a closing bracket.

**Comment by dylandevei** [comment]
Screen Shot 2022-03-25 at 10 57 41 PM
made the necessary revisions to code and moved return to their own lines

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**